velox/core/QueryConfig.h

```diff
@@ -589,6 +592,18 @@ class QueryConfig {
         kMaxBits, get<uint8_t>(kJoinSpillPartitionBits, kDefaultBits));
   }
 
+  /// Returns the number of bits used to calculate the spilling partition
+  /// number for RowNumber. The number of spilling partitions will be power of
+  /// two.
+  ///
+  /// NOTE: as for now, we only support up to 8-way spill partitioning.
```

Comment:
spilling -> spill

Reply:
Fixed, and fix the comments of `joinSpillPartitionBits()` too.